_posts/2019-10-??-wprowadzenie-do-zed-attack-proxy.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Wprowadzenie do Zed Attack Proxy
+published: false
+date:      2019-09-30 03:00:00 +0100
+author:    amak
+tags:
+  - programming
+  - pentest
+  - pentesting
+  - tool
+  - OWASP
+  - security
+---
+
+ZAP (Zed Attack Proxy) jest opensourcowym narzędziem tworzonym przez organizację OWASP wspomagającym testy penetracyjne, które służy do znajdowania podatności bezpieczeństwa w aplikacjach
+webowych. Działa na zasadzie proxy, dzięki czemu pozwala nie tylko na podglądanie żądań wysyłanych do serwera aplikacji i odpowiedzi z serwera otrzymywanych, ale daje również
+możliwość debugowania, modyfikowania oraz wysyłania własnych żądań. Jest narzędziem dostosowanym do obsługi HTTP, oferuje prosty sposób na rozszyfrowanie HTTPS poprzez dodanie własnego
+certyfikatu do przeglądarki. Dostarcza automatyczne skanery, jak również narzędzia pomagające manualnie testować aplikację.
+
+## Instalacja oraz konfiguracja
+
+Link do pobrania:
+
+https://github.com/zaproxy/zaproxy/wiki/Downloads
```

Comment:
1. Fajnie jakby to było faktycznie hiperłącze (tak żeby można było kliknąć)
2. Mogłoby być w tej samej linijce co "Link do pobrania:"